fileLogger.ts

```diff
@@ -1,25 +1,35 @@
-import winston, { createLogger } from "winston";
+import winston, { createLogger, Logger } from "winston";
```

Comment:
hindi ba to magcoconflict sa Logger sa logger.ts?

Reply:
d naman po since nasa kabilang file naman yung Logger class mo po mismo